drivers/gps_sim/Kconfig

```diff
@@ -7,6 +7,7 @@
 
 menuconfig GPS_SIM
 	bool "GPS simulator"
+	imply FP_SHARING
```

Comment:
Looks good; should we also add if FLOAT here? Cause FP_SHARING has no meaning without it.
Same in the SENSOR_SIM below

Reply:
True, thanks!